ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -56,8 +56,10 @@ mac_address_table:
 # Optional profiles to apply on SVI interfaces
 # Each profile can support all or some of the following keys according to your own needs.
 # Keys are the same used under SVI.
+# svi_profiles can refer to another svi_profiles to inherit settings in up to two levels (svi->profile->parent_profile).
 svi_profiles:
   < profile_name >:
+    parent_profile: < port_profile_name >
```

Comment:
``parent_profile: < svi_profile_name >`` will describe it better

Reply:
Thanks, will be fixed shortly.